project/src/views/new-evaluations/newEvaluationsViews.js

```diff
@@ -24,356 +24,98 @@ const CandidateDetailsForm = function() {
     `;
 }
 
-const TechnicalLevelPicker = function() {
+const TechnicalLevelHeader = function(options){
+    const headings = options.headings.map((el) => `<th>${el}</th>`);
+    return `
+    <tr>
+       ${headings.join('')}
+    </tr>
+    `;
+}
+
+const TechnicalLevelPickerHeaders = function(options = {}){
+    const str = options.map((el) => `<th>${el}</th>`);
+    return str;
+}
+
+const TechnicalLevelPickerColumns = function(options={}){
+    const inputs = options.map((inp) => {
+        const partialInputs = inp.inputLevels.map((lev) => `<input type="radio" id="${lev}" name="level"/>`);
+        return `<td>${partialInputs.join('')} </td>`;
+        });
+        return inputs;
+
+}
+
+const TechnicalLevelPicker = function(options={}) {
+    const headers = TechnicalLevelPickerHeaders(options.headers);
+    const columns = TechnicalLevelPickerColumns(options.columnData);  
     return `
     <section>
-        <h2 class="h2-skin">Technical Level</h2>
+        <h2 class="h2-skin">${options.title}</h2>
         <table>
         <tr>
-            <th>Trainee</th>
-            <th>Junior</th>
-            <th>Middle</th>
-            <th>Senior</th>
+            ${headers.join('')}
         </tr>
         <tr class="border-lists-skin">
-            <td>
-            <input type="radio" id="T"/>
-            </td>
-            <td>
-            <input type="radio" id="J1"/>
-            <input type="radio" id="J2"/>
-            <input type="radio" id="J3"/>
-            </td>
-            <td>
-            <input type="radio" id="M1"/>
-            <input type="radio" id="M2"/>
-            <input type="radio" id="M3"/>
-            </td>
-            <td>
-            <input type="radio" id="S1"/>
-            <input type="radio" id="S2"/>
-            <input type="radio" id="S3"/>
-            </td>
+            ${columns.join('')}
         </tr>
         </table>
     </section>
     `;
 }
-
-const Textarea = function(options){
+const Textarea = function(options = {}){
     return `
     <section>
-        <h2 class="h2-skin">Should the candidate be hired?</h2>
-        <textarea rows="6" placeholder=bla></textarea>
-    </section>
-
-    <section>
-        <h2  class="h2-skin">General Impression</h2>
-        <textarea rows="6" required placeholder="*required"></textarea>
-    </section>
+        <h2 class="h2-skin">${options.title}</h2>
```

Comment:
Can we get away from using tag names in class names, pretty please? :pray:

Reply:
Yes, we can :)